UI/src/components/widgets/build/directives/averageBuildDuration.html

```diff
@@ -2,7 +2,7 @@
     <div class="widget-heading">
         Average build duration
     </div>
-    <div hide-empty="['{{ buildView.buildDurationData }}']">
+    <div hide-empty="{{ buildView.buildDurationData.series }}">
```

Comment:
@kheinz04  check for undefined on buildView.buildDurationData

Reply:
undefined assigned automatically through angular